scripts/inner/kubestellar-kube-bind

```diff
@@ -86,21 +74,18 @@ if kubectl get crd $resource_to_bind.edge.kubestellar.io &> /dev/null; then
     exit 0
 fi
 
-if [[ ! -d $log_dir ]]; then
-    mkdir -p $log_dir
-fi
-
-if [ "$start_konnector" == true ]
-then
+if pgrep konnector | xargs ps e -p 2>/dev/null | grep "KB_CONSUMER=$space_name" &> /dev/null; then
```

Comment:
(1) What if one space name is a prefix of another? You could add a terminal character that is valid in an envar value but not in a Space name.

(2) I think you need some `w`s on that `ps` command:
```
mspreitz@mjs-dev7a:~$ ps e -p 1146462
    PID TTY      STAT   TIME COMMAND
1146462 pts/0    Ss     0:00 -bash LANG=C.UTF-8 USER=mspreitz LOGNAME=mspreitz H

mspreitz@mjs-dev7a:~$ ps ew -p 1146462
    PID TTY      STAT   TIME COMMAND
1146462 pts/0    Ss     0:00 -bash LANG=C.UTF-8 USER=mspreitz LOGNAME=mspreitz HOME=/home/mspreitz PATH=/usr/local/sbin:/usr/local/b

mspreitz@mjs-dev7a:~$ ps eww -p 1146462
    PID TTY      STAT   TIME COMMAND
1146462 pts/0    Ss     0:00 -bash LANG=C.UTF-8 USER=mspreitz LOGNAME=mspreitz HOME=/home/mspreitz PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin SHELL=/bin/bash TERM=xterm-256color XDG_SESSION_ID=531 XDG_RUNTIME_DIR=/run/user/1002 DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1002/bus XDG_SESSION_TYPE=tty XDG_SESSION_CLASS=user MOTD_SHOWN=pam SSH_CLIENT=9.67.15.143 58072 22 SSH_CONNECTION=9.67.15.143 58072 9.59.199.171 22 SSH_TTY=/dev/pts/0
```

On Ubuntu and Darwin, two `w` means unlimited width.

(3) On Darwin, you need `E` rather than `e` to get the environment displayed.